script/sync

```diff
@@ -134,6 +134,30 @@ def sync_ruby(version)
   puts "done."
 end
 
+def sync_node(version)
+  s3_key = "nodes/Darwin/#{s3_cellar}#{os}/#{version}.tar.bz2"
```

Comment:
He mike, sorry... but I don't really understand what I'm doing nor how to do it.... can you expand of what are we checking?? thanks! 😁